### DISCUSSION TITLE
🎗️我也想为项目做出贡献

### DISCUSSION THREAD

**Discussion by nekocatso** [misaka-org/Chaoxing-MiniProgram#5] [General]
你好,作为一名大学生,我深受你们的开源精神所感染,我能作为项目贡献的一份子吗?

**Comment by Misaka-1314** [comment]
与其说是开源精神，不如说是想水点star + 引流

**Reply by Suntudou** [reply]
已star，（doge讲座必备品啊

**Comment by Misaka-1314** [comment]
你想做些什么？

注：以下接口不会开源，也不会被合并
+ 滑块验证码
+ 自动获取位置签到的位置
+ 自动获取签到码、手势
+ 随机借用已签同学的照片
